### DISCUSSION TITLE
Mage Conjured Refreshment Cataclysm Classic

### DISCUSSION THREAD

**Discussion by roweboy1974** [HgAlexx/Buffet#70] [Q&A]
I can't seem to find a macro that works for conjured refreshment on my mage in Cata Classic. Any help would be greatly appreciated.

**Comment by HgAlexx** [comment]
Hello,

Is this the related item ? https://www.wowhead.com/cata/item=43518/conjured-mana-pie

What is your client language ?

Please give the the output of the following command (replace [itemLink] with a link to the item):
/buffet debug [itemLink]

Thank you.

**Comment by HgAlexx** [comment]
Nevermind, fixed in version 110.
